src/main/java/games/strategy/util/TimeManager.java

```diff
@@ -1,20 +1,54 @@
 package games.strategy.util;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.SimpleTimeZone;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.FormatStyle;
 
 public class TimeManager {
   /**
-   * Replacement for Date.toGMTString();
+   * Prints an {@link Instant} localized, with all details
```

Comment:
Checkstyle: "First sentence of Javadoc is incomplete (period is missing) or not present."